BeatSaverVoting/Utilities/BeatSaverAPIResult.cs

```diff
@@ -38,13 +38,13 @@ public bool Compare(Song compareTo)
             return compareTo.hash == hash;
         }
 
-        public Song(CustomPreviewBeatmapLevel data)
+        public Song(BeatmapLevel data)
```

Comment:
This isn't used as far as I can see. Should remove it rather than setting stuff to empty strings.